src/utils/request.ts

```diff
@@ -42,6 +45,14 @@ function getOffsetEndpoint(config: Config, featureCount: number): string {
   );
 }
 
+function getOffsetEndpointNGD(config: Config, featureCount: number): string {
+  const limit = Math.min(
+    config.paging.limitValue - config.paging.startValue - featureCount,
+    100
+  );
+  return config.url + "&offset=" + config.paging.position + "&limit=" + limit;
```

Comment:
Should we use URLSearchParams instead of strings to make this more readable

Reply:
In such a simple case, I'm not sure this is more readable - search params version looks like this:

`return (
    config.url +
    new URLSearchParams({ offset: config.paging.position.toString(), limit: limit.toString() }).toString()
  );`